packages/driver/src/cypress/error_messages.ts

```diff
@@ -1936,7 +1957,7 @@ export default {
           > ${subjectChainToString(obj.subjectChain)}`
     },
     state_subject_deprecated: {
-      message: `${cmd('state', '\'subject\'')} has been deprecated and will be removed in a future release. Consider migrating to ${cmd('currentSubject')} instead.`,
+      message: `${cmd('state', '\'subject\'')} has been deprecated and will be removed in a future release. Consider migrating to ${cmd('subject')} instead.`,
```

Comment:
Changing from cy.currentSubejct to cy.subject is breaking. Do we need to make a err message for those using who may have already updated to use cy.currentSubject to say it's been removed & they need to use cy.subject instead?

Reply:
I'd prefer to mention it in the changelog as breaking and not leave behind any cruft.

As far as I know, the only consumer of `state('subject')` (other than us) is cypress-testing-library, and they haven't updated yet.